README.md

```diff
@@ -204,8 +197,21 @@ _* Mono JIT compiler (tested with version 4.6.0)_
   nunit-console src/Tests/bin/Release/Tests.dll --config=Release
   ```
 
+## Publish to the Nearmap NuGet Repository
+
+Build it first. See above.
 
+### on Linux, OS X
 
+1. Build the NuGet package
+  ```
+  mono lib/NuGet.exe pack src/StatsdClient/StatsdClient.nuspec -Version '<version>' -BasePath <path>/src/StatsdClient -IncludeReferencedProjects
```

Comment:
Is `-Version` required when `<version>` is already in the `.nuspec`?

Reply:
Oh yes. Much better.